warehouse/query-core/src/test/java/datawave/query/index/lookup/RangeStreamTest.java

```diff
@@ -1225,64 +1210,21 @@ public void testDropTwoPredicates() throws Exception {
         helper.addFields(Arrays.asList("FOO", "LAUGH"));
         
         // Create expected ranges verbosely, so it is obvious which shards contribute to the results.
```

Comment:
This comments may no longer be applicable

Reply:
They are not applicable anymore, will remove in the next commit.